UA-NetStandard.sln

```diff
@@ -51,23 +51,75 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NetCoreConsoleClient", "Sam
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NetCoreConsoleServer", "SampleApplications\Samples\NetCoreConsoleServer\NetCoreConsoleServer.csproj", "{F7880521-6662-43A0-B28E-219AA8D28A49}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "XamarinClient.Android", "SampleApplications\Samples\XamarinClient\XamarinClient.Android\XamarinClient.Android.csproj", "{F7FE6E6A-A37D-48AF-84D5-60727679D973}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "XamarinClient.iOS", "SampleApplications\Samples\XamarinClient\XamarinClient.iOS\XamarinClient.iOS.csproj", "{9910F67A-D3EC-41DC-B67B-1E16A5E1338F}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "XamarinClient.UWP", "SampleApplications\Samples\XamarinClient\XamarinClient.UWP\XamarinClient.UWP.csproj", "{F71BCE6C-4071-42A8-894F-185AFD3427A4}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "XamarinClient", "SampleApplications\Samples\XamarinClient\XamarinClient\XamarinClient.csproj", "{1C3C2B89-AC98-4EDA-96EC-42C2547D8A83}"
```

Comment:
Could you revert the .Net Standard solution file? There is still rubbish from the xamarin project.

16.48s
$ dotnet restore "UA-NetStandard.sln"

/home/travis/build/OPCFoundation/UA-.NETStandard/SampleApplications/Samples/XamarinClient/XamarinClient.Android/XamarinClient.Android.csproj(94,3): error MSB4019: The imported project "/usr/share/dotnet/sdk/2.0.0/Xamarin/Android/Xamarin.Android.CSharp.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.

/home/travis/build/OPCFoundation/UA-.NETStandard/SampleApplications/Samples/XamarinClient/XamarinClient.Android/XamarinClient.Android.csproj : warning NU1503: Skipping restore for project '/home/travis/build/OPCFoundation/UA-.NETStandard/SampleApplications/Samples/XamarinClient/XamarinClient.Android/XamarinClient.Android.csproj'. The project file may be invalid or missing targets required for restore. [/home/travis/build/OPCFoundation/UA-.NETStandard/UA-NetStandard.sln]